lldb/test/API/lang/swift/cxx_interop/forward/test-cxx-class/TestClass.py

```diff
@@ -0,0 +1,23 @@
+
+"""
+Test that a C++ class is visible in Swift.
+"""
+import lldb
+from lldbsuite.test.lldbtest import *
+from lldbsuite.test.decorators import *
+import lldbsuite.test.lldbutil as lldbutil
+import os
+import unittest2
+
+
+class TestClass(TestBase):
+
+    @swiftTest
+    def test_class(self):
+        self.build()
+        self.runCmd('setting set target.experimental.swift-enable-cxx-interop true')
+        target, process, thread, bkpt = lldbutil.run_to_source_breakpoint(
+            self, 'Set breakpoint here', lldb.SBFileSpec('main.swift'))
+
+        self.expect('v x', substrs=['CxxStruct', 'a1', '1', 'a2', '2', 'a3', '3'])
```

Comment:
Maybe [lldbutil.check_expression/variable](https://github.com/apple/llvm-project/blob/a3a1b62aa5f606a7afae17db5a528e566474bdba/lldb/packages/Python/lldbsuite/test/lldbutil.py#L1460)?

Reply:
Since I want to check multiple strings, I think expect is easier, unless there's a reason to use chech_expression/variable I'm not seeing. I've updated the test to make it cleared what are the fields and variables we're looking for.